clang/test/CIR/Transforms/loop.cir

```diff
@@ -26,4 +26,45 @@ module {
 // CHECK:    cir.br ^bb[[#COND:]]
 // CHECK:  ^bb[[#EXIT]]:
 // CHECK:    cir.return
+// CHECK:  }
+
+  // Test while cir.loop operation lowering.
+  cir.func @testWhile(%arg0 : !cir.bool) {
+    cir.while {
+      cir.condition(%arg0)
+    } do {
+      cir.yield
+    }
+    cir.return
+  }
+
+// CHECK:  cir.func @testWhile(%arg0: !cir.bool) {
+// CHECK:    cir.br ^bb[[#COND:]]
```

Comment:
Is this just doing a 'match all' with the named capture?  Slight preference for a '*' for the regex match instead.

Reply:
It's specifically matching a number.

Reply:
Huh, neat!  I have never seen that feature before.